Events/Ch17.event

```diff
@@ -167,6 +167,7 @@ ENUN
 LOAD2 0x1 Foes3
 ENUN
 LOAD2 0x1 Foes4
+GotoPrepScreen
```

Comment:
Are no changes to units loading necessary? At the very least, ally loading is automatic once the prep screen starts (the one pointed to by the pointer table unit pointers).

I just feel like there can be some cleanup in this opening event. Also the new unit is silently added to the party, not loaded on screen, which may or may not be intended behavior.

We could also defer refactoring the events until a later date. Feel free to discuss on Discord.